### PR TITLE
Updating installation guilde

### PR DESCRIPTION
**Pull request by anpaz-msft**
Simplifying installation guide, to make it include Python and Jupyter and to make it simpler for non-VS users.

### CODE REVIEW

File: articles/install-guide/csharp.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started with Python and Q#
+author: cpalmer
```
Comment (cpalmer2020) [inline articles/install-guide/csharp.md]: anpaz-msft

File: articles/install-guide/csharp.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+uid: microsoft.quantum.install.csharp
+ms.author: cpalmer@microsoft.com
```
Comment (cpalmer2020) [inline articles/install-guide/csharp.md]: anpaz-msft

File: articles/install-guide/csharp.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+C:\>cd myApp
+
+C:\Repos\myApp>dotnet run
+Hello quantum world!
+```
+
+
+## Usage ##
+
+Take a look at the [quickstart guide](xref:microsoft.quantum.write-program) for information on how to write
```
Comment (cpalmer2020) [inline articles/install-guide/csharp.md]: link doesn't work?

File: articles/install-guide/index.md [hunk 4]
```diff
@@ -1,40 +1,89 @@
...
-<tr>
-<td>
-[<img src="../media/install-guide-select-vs-2017.png" width="90%">](xref:microsoft.quantum.install.vs-2017)
-<div class="nextstepaction">
-[Install on Visual Studio 2017 (Windows 10)](xref:microsoft.quantum.install.vs-2017)
-</div>
-</td>
+Be mindful that all changes you do to notebooks launched this way cannot be saved. In order 
+to create new notebooks or write new quantum applications you will need to install 
+IQ# to your local machine.
```
Comment (cpalmer2020) [inline articles/install-guide/index.md]: the IQ# Jupyter Kernel

File: articles/install-guide/index.md [hunk 5]
```diff
@@ -1,40 +1,89 @@
...
+
+Applications developed with Microsoft's Quantum Development Kit typically consists of two parts:
+1. One or more quantum algorithms, implemented using the Q# quantum programming language.
+2. A classical program, implemented in a classical programming language like Python or C#, 
+  that serves as the main entry point and will invoke Q# operations 
+  when it wants to execute a quantum algorithm.
+
+Once you have installed IQ#, these articles explain how to enable interoperability between Q# and other classical languages:
+
+* [C# getting started](xref:microsoft.quantum.install.csharp)
```
Comment (cpalmer2020) [inline articles/install-guide/index.md]: iQSharp is not required for C#, is it?

File: articles/install-guide/index.md (hunk 4 above)
Comment (anpaz-msft) [reply articles/install-guide/index.md]: `(xref)` works if viewed from the preview site (i.e. https://review.docs.microsoft.com/en-us/quantum/install-guide/csharp?view=qsharp-preview&branch=anpaz%2Fgetting-started), but don't work when using markdown preview from github.

File: articles/install-guide/csharp.md (hunk 3 above)
Comment (anpaz-msft) [reply articles/install-guide/csharp.md]: will work from review.docs.microsoft.com

File: articles/install-guide/index.md (hunk 5 above)
Comment (anpaz-msft) [reply articles/install-guide/index.md]: Talked about this. Using this as a way to verify that Q# is ready to go and to give people something to install.

**Pull request merged**